app/lib/screens/home/home.dart

```diff
@@ -1,37 +1,23 @@
-import 'package:app/navbar/navigation.dart';
-import 'package:app/sidebar/sidebar.dart';
+import 'package:app/navigations/nav_bar/navigation.dart';
+import 'package:app/navigations/side_bar/sidebar.dart';
 import 'package:flutter/material.dart';
+import 'package:app/providers/navigation_provider.dart';
+import 'package:provider/provider.dart';
 
-class HomePage extends StatelessWidget {
+class HomeScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    print('Application starting....');
```

Comment:
Is it common to have these print statements in the final result? Maybe these are placeholders for like the buffering icon we intend in #99 ?

Reply:
These printing statements is for me to check if widget is build. In the future, we should use logs similar to .NET when I'm developing web apps. I'm pretty sure flutter has its own logging mechanism or even a plugin for it.

Reply:
Yup, just noticed it in the build output now. We can add a task for logs later (when we come upon it)